src/base-input.js

```diff
@@ -596,37 +602,50 @@ export default class BaseInput extends LitElement {
 
     // eslint-disable-next-line sort-vars, one-var
     const creditCardTypes = [
+      {
+        name: 'Commercial',
+        regex: /^(?<num>2)\d{0}/u,
+        formatMinLength: 8,
+        customValidationMessage: defaultCustomValidationMessage
+      },
```

Comment:
See comment below for why this is necessary

```suggestion
        customValidationMessage: defaultCustomValidationMessage,
        cardIcon: 'credit-card'
      },
```